doit/cmdparse.py

```diff
@@ -333,7 +333,7 @@ def parse_only(self, in_args, params=None):
         return params, args
 
 
-    def parse(self, in_args):
```

Comment:
It is NOT ok to make any changes on `cmdparse.py` module to fix this issue.
`cmdparse` already has support to handle different sources of values (although not explicitly).
You should use `overwrite_defaults()`, see other comment...

Reply:
I've reverted modifications to this file.